src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,67 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
 
-    public void getStatistic(String fromFileName, String toFileName) {
+    private String readFromFile(String fromFileName) {
+        fromFileName = "orange.csv";
+        File file = new File(fromFileName);
+        StringBuilder builder = new StringBuilder();
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(file))) {
+            int value = bufferedReader.read();
+            while (value != -1) {
+                builder.append((char) value);
+                value = bufferedReader.read();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read file" + file, e);
+        }
+        return builder.toString();
+    }
+
+    private String createReport(String fromFile) {
+        String[] split = readFromFile(fromFile).replaceAll("(\r\n|\n)", " ").split(" ");
```

Comment:
```suggestion
    private String createReport(String rows) {
        String[] split = rows.split("\\s+");
```
навіщо ще раз зчитувати з файлу, якщо ти вже передаєш сюди готовий контент